core_backend/app/urgency_detection/routers.py

```diff
@@ -88,17 +105,21 @@ async def cosine_distance_classifier(
 
 @urgency_classifier
 async def llm_entailment_classifier(
-    asession: AsyncSession,
+    user_id: str,
     urgency_query: UrgencyQuery,
+    asession: AsyncSession,
 ) -> UrgencyResponse:
     """
     Classify the urgency of a text message using LLM entailment
     """
-
-    rules = await get_urgency_rules_from_db(asession)
+    rules = await get_urgency_rules_from_db(user_id=user_id, asession=asession)
     tasks = []
     for rule in rules:
-        tasks.append(detect_urgency(rule.urgency_rule_text, urgency_query.message_text))
+        tasks.append(
```

Comment:
whoa. What is your max line length set to?

Reply:
Pretty sure it's 79, the [PEP8 standard](https://peps.python.org/pep-0008/#maximum-line-length)

Reply:
We use 88!
https://docs.google.com/document/d/1fIDbGrUdBKP0qnSkVVNpwjYmB1e8FW2n9Uzei_9v7k8/edit

Reply:
Fixed in my configs :)